CMakeLists.txt

```diff
@@ -25,16 +28,12 @@ if(WITH_TWEETNACL)
   set(TWEETNACL_SOURCES
     tweetnacl/src/tweetnacl.c
     )
-  if(WIN32)
-  else()
+
+  if(NOT WIN32)
     list(APPEND TWEETNACL_SOURCES tweetnacl/contrib/randombytes/devurandom.c)
```

Comment:
So this breaks windows now when not using libsodium.

randombytes is an unresolved external, presumably tweetnacl has never worked on windows but now we are forcing its use.